### PR TITLE
Issue 265 document maintenance

### LINKED ISSUE #265: Document maintenance practices of the repo in MAINTAINING.md or similar
The repo and the community seem to be accreting some conventional practices, e.g. the Zenodo webhook that generates a DOI for releases (from issue #136), and possibly others. We should collect this info into a single file which describes our solutions to various logistical issues that have arisen.

### PR DESCRIPTION
**Pull request by reid-a**
Initial MAINTENANCE.md file with the initially-identified content.
Closes #265.
Subsequent additions can be separate issues.

### CODE REVIEW

**Review by mikerenfro — APPROVED** [review]

File: MAINTENANCE.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+
+[hpc-intro]: https://github.com/carpentries-incubator/hpc-intro
+[incubator-base]: https://github.com/carpentries-incubator
+[project-github]: https://github.com/hpc-carpentry
+[coord-repo]: https://github.com/hpc-carpentry/coordination
+[mainsite-repo]: https://github.com/hpc-carpentry/hpc-carpentry.org
+[swc-slack]: https://swcarpentry.slack.com
+[hpc-slack]: https://swcarpentry.slack.com/#hpc-carpentry
+[slakc-invite]: https://swc-slack-invite.herokuapp.com
```
Comment (mikerenfro) [inline MAINTENANCE.md:81]: Replace with slakc-invite with slack-invite

**Review by mikerenfro — CHANGES_REQUESTED** [review]

File: MAINTENANCE.md (hunk 1 above)
Comment (reid-a) [reply MAINTENANCE.md]: Fixed, thanks! 
There's also build weirdness, see issue #293 .

**Review by tkphd — APPROVED** [review]

File: MAINTENANCE.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+account to contribute.  In addition, in order to get write access to
+the repo, you will need to become a member of the HPC Intro
+Maintainers team, which lives inside the "carpentries-incubator"
+[project][incubator-base].
+
+See also the instructions for [contributing](CONTRIBUTING.md).
+
+#### HPC-Carpentry Project
+
+There is also an HPC Carpentry [Github project][project-github], which
```
Comment (tkphd) [inline MAINTENANCE.md:27]: Github -> GitHub

File: MAINTENANCE.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+has other HPC-related lessons repos in its scope, and separate teams
+of maintainers and administrators.
+
+In particular, there is a "coordination" [repo][coord-repo], which is
```
Comment (tkphd) [inline MAINTENANCE.md:31]: repo -> repository

**Review by mikerenfro — APPROVED** [review]

File: MAINTENANCE.md [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+[swcarpentry.slack.com][swc-slack]. You can go directly to the HPC
+Carpentry sub-channel at [#hpc-carpentry][hpc-slack]. These are the
+things to bookmark -- once you go there, the Slack engine redirects
+you to particular views, so cutting/pasting URLs out of the browser's
+location bar is likely to be disappointing.
+
+Members of existing swcarpentry slack channels can find the channel by
+typing "hpc-carpentry" into the slack search box.
+
+To become a member, you can trigger in invitation from [here][slack-invite].
```
Comment (tkphd) [inline MAINTENANCE.md:51]: Move "how to join" to the top.

File: MAINTENANCE.md [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+
+### E-mail:
+
+The top-level site is on [Topicbox][topicbox]. There are two
+HPC-specific lists, "discuss-hpc" and "maintainers-hpc". The system
+does not use passwords, log-ins are via a verification code sent to
+your e-mail. I don't recall how to join the lists, my recollection is
```
Comment (tkphd) [inline MAINTENANCE.md:58]: Your recollection is correct: anyone can join any list. Please rephrase.

**Review by tkphd — CHANGES_REQUESTED** [review]

**Pull request merged**